lib/classes/movie.rb

```diff
@@ -10,11 +10,14 @@ def can_be_archived?
     super || @silet ? true : false
   end
 
-  def to_json(*args)
+  def to_json()
     {
-      JSON.create_id => self.class.name,
       'id' => id,
+      'genre_id' => @genre.id,
+      'author_id' => @author.id,
+      'source_id' => @source.id,
+      'label_id' => @label.id,
       'silet' => @silet
-    }.to_json(*args)
+    }.to_json()
   end
```

Comment:
### Hello, @Trictonicmp in this file you need to keep this: 
> JSON.create_id => self.class.name 

**in the to_json method, because that's the way ruby knows which class is used to create the object stored in the JSON file**

Reply:
Oh, all right! Thank you so much, I'll update the code